python/friesian/dev/test/add_type_hint.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2022 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# set -ex
+# . `dirname $0`/../prepare_env.sh
+
+cd "`dirname $0`"
+export MT_DB_PATH="$(pwd)/friesian_hint.sqlite3"
```

Comment:
where's this file?

Reply:
> where's this file?

it specifies the database path will be used by monkeytype.